src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerArgumentBinder.cs

```diff
@@ -189,13 +189,13 @@ public class ControllerArgumentBinder : IControllerArgumentBinder
             await binder.BindModelAsync(modelBindingContext);
 
             var modelBindingResult = modelBindingContext.Result;
-            if (modelBindingResult != null && modelBindingResult.Value.IsModelSet)
+            if (modelBindingResult != null && modelBindingResult.IsModelSet)
```

Comment:
:+1: - will do a find all references for `.Result` in the product and double check that I didn't miss any.